test/e2e/common/application_autoscaling_utils.py

```diff
@@ -0,0 +1,71 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
can rename file to utils.py?

Reply:
I think this name is more appropriate since this bootstraps AA resources using boto3 for the tests ?